app/config/custom-inflector-rules.js

```diff
@@ -1,39 +1,34 @@
-import Inflector from 'ember-inflector';
+import { plural, singular, irregular } from '@ember-data/request-utils/string';
 
-const inflector = Inflector.inflector;
+plural(/$/, 'en');
```

Comment:
Not strictly needed for the update, but a nice improvement could be to remove all these rules to configure the dutch pluralization logic, and simply add exceptions for the remaining dutch models.

The end goal is to have everything in English.

Similar commit in Loket: https://github.com/lblod/frontend-loket/commit/7318dd69ea927493d8d06df4c6200f24d14b06dd

It seems there are only 3 dutch models in LPDC: `bestuurseenheid-classificatie-code`, `bestuurseenheid` and `gebruiker`. So it should be a very short list, besides some other exceptions maybe.

Anyway, doesn't need to be part of this PR, but would be nice if you have the time.